mozregression/launchers.py

```diff
@@ -163,6 +163,8 @@ def _install(self, dest):
             raise
 
     def _start(self, profile=None, addons=(), cmdargs=(), preferences=None):
+        if not os.path.exists(profile):  
```

Comment:
profile can be None - so it should be something like "if profile and not os.path.exists(profile)".